packages/app-builder-lib/src/node-module-collector/pnpmNodeModulesCollector.ts

```diff
@@ -28,54 +29,51 @@ export class PnpmNodeModulesCollector extends NodeModulesCollector<PnpmDependenc
     return ["list", "--prod", "--json", "--depth", "Infinity"]
   }
 
-  protected extractRelevantData(npmTree: PnpmDependency): PnpmDependency {
-    const tree = super.extractRelevantData(npmTree)
-    return {
-      ...tree,
-      optionalDependencies: this.extractInternal(npmTree.optionalDependencies),
-    }
-  }
+  extractProductionDependencyGraph(tree: PnpmDependency, dependencyId: string): void {
+    if (this.productionGraph[dependencyId]) return
```

Comment:
Minor, multi-line `if` codeestyle for `return`

Reply:
Fixed